packages/react-core/src/demos/RTL/RTL.md

```diff
@@ -26,6 +26,6 @@ This demonstrates how the UI adapts to the writing mode of the page.
 
 ### Paginated table
 
-```js file="./examples/PaginatedTable.jsx" isFullscreen
+```ts file="./examples/PaginatedTable.tsx" isFullscreen
```

Comment:
```suggestion
```js file="./examples/PaginatedTable.tsx" isFullscreen
```

I think this will fix the docs build issue here, there's something weird going on in our documentation framework when we declare a TS codeblock, but TS files seem to import into JS blocks just fine.

Reply:
It works. Thanks for the suggestion.